### DISCUSSION TITLE
Subdomain

### DISCUSSION THREAD

**Discussion by remotejob** [hktalent/scan4all#113] [Q&A] [answered]
SSL Analysis ---> **Subdomain** 

It look like not active yet?

**Comment by hktalent** [comment, marked as answer]
1、config to(config.json):
"EnableKsubdomain": true,
"EnableSubfinder": true,
"ParseSSl": true,

EnableSubfinder=true
ParseSSl=true
noScan=false

2、domain start "*.",eg:
*.gov.com

EnableSubfinder default is false

2、
ParseSSl=true EnableSubfinder=true ./scan4all -host xxx.com

**Reply by remotejob** [reply]
I try to reduce action only for subnames enumeration my config.json

`"UrlPrecise": false,
  "ParseSSl": true,
  "EnableSubfinder": true,
  "EnableHoneyportDetection": false,
  "EnableKsubdomain": true,
  "KsubdomainRegxp": "([0-9a-zA-Z\\-]+\\.[0-9a-zA-Z\\-]+)$",
  "naabu_dns": {},
  "naabu": {"TopPorts": "1000","ScanAllIPS": false,"Threads": 50,"EnableProgressBar": false},
  "priorityNmap": false,
  "noScan": false,
  "enableMultNuclei": false,
  "enableNuclei": false,
  "nuclei": {
    "Severities":  [5,4,3],
    "RateLimit": 150,
    "ExcludeTags": ["fuzz"],
    "BulkSize":64,
    "TemplateThreads": 64,
    "HeadlessBulkSize": 10,
    "DisableRedirects": false,
    "EnableProgressBar": false,
    "HeadlessTemplateThreads": 10,
    "ReportingConfig": "config/nuclei_esConfig.yaml"
  },
  "enablDevDebug": false,
  "enablEmbedYaml": false,
  "enablFileFuzz": false,
  "httpx": {
    "Pipeline": false,
    "HTTP2Probe": false,
    "VHost": false,
    "CSPProbe": false,
    "TLSProbe": false,
    "TechDetect": false,
    "FollowRedirects": false,
    "MaxRedirects": 3
  },
  "enableEsSv": false,
  "CheckWeakPassword": false,
  "esthread": 8,
  "hydrathread": 64,
  "Fuzzthreads": 32,
  "enableFingerTitleHeaderMd5Hex": false,
  "Cookie": "",
  "esUrl": "http://127.0.0.1:9200/%s_index/_doc/%s",
  "Exploit":{
    "Path": "./config/poc/",
    "Logs": "./logs/errror.log"
  },
  "enableWebScan": false
}

`

./scan4all -l  domainsalive.txt -v -o output/domain.txt

but after first domain
ERR] Could not verbose results for tonttimyynti.fi: short write
FTL] Could not run enumeration: short write

Question: what will be the best config.json for only subdomains enumeration task?

**Reply by hktalent** [reply]
set 
"UrlPrecise": true,

**Reply by remotejob** [reply]
Don't help 
[ERR] Could not verbose results for tonttimyynti.fi: short write
[FTL] Could not run enumeration: short write

**Comment by hktalent** [comment]
1、config to(config.json):
EnableSubfinder=true
ParseSSl=true
noScan=false
2、domain start "*.",eg:
*.gov.com

**Comment by hktalent** [comment]
$ cat list.txt 
```
www.baidu.com
www.163.com
```
$ EnableSubfinder=true ParseSSl=true noScan=false ./scan4all -l list.txt -v

<img width="1108" alt="image" src="https://user-images.githubusercontent.com/18223385/193034424-952f5aa2-bfea-4e40-b288-6105cbc9de04.png">

<img width="497" alt="image" src="https://user-images.githubusercontent.com/18223385/193034398-1747cb19-ba71-41b2-94b1-a0b8cff42e9c.png">
